### ISSUE TITLE
Update CDEPS ESMF Config format

### ISSUE THREAD

**Issue by uturuncoglu** [ufs-community/ufs-weather-model#1548]
## Description
The stream definition config file is diverged between XML and ESMF Config options. This needs to be fixed.

## Solution
The `shr_stream_init_from_esmfconfig()` call under `CDEPS/streams/dshr_stream_mod.F90` needs to be updated to reflect recent changes in CDEPS side to make both XML and ESMF Config formats are consistent.

## Alternatives
N/A

## Related to
This is also explained in the https://github.com/ESCOMP/CDEPS/issues/200

**Comment by uturuncoglu** [comment]
@junwang-noaa @binli2337 I am currently working on CDEPS documentation as a part of JTTI land project and I realized that the keys used in XML and ESMF Config formats to define data stream configurations are diverged. The potential changes in `shr_stream_init_from_esmfconfig()` will probably affect the parameter files under `tests/parm` directory. I just wonder that you are fine with those changes. I think we might also support old keys, which might not lead any change in the RT configuration files. Anyway, let me know what you think?

**Comment by junwang-noaa** [comment]
@uturuncoglu I think it is good to have consistent XML and ESMF config formats. Really appreciate it if you can sync UFS CDEPS branch with the latest code updates and make changes in the files under UFS WM tests/parm. Thanks

**Comment by uturuncoglu** [comment]
@junwang-noaa Sure. I could include them to my next PR related with land coupling.

**Comment by DeniseWorthen** [comment]
This issue is almost a year old. Is it fixed?

**Comment by uturuncoglu** [comment]
@DeniseWorthen No. I could not find time to fix it. I think this also involves some work with the CDEPS specific RT template files.

**Comment by DeniseWorthen** [comment]
@uturuncoglu OK, thanks for the update.

**Comment by gspetro-NOAA** [comment]
@uturuncoglu @NickSzapiro-NOAA Has this issue been resolved via CDEPS PR #66 and UFS WM PR #2469 ? Or is there additional work to be done?

**Comment by NickSzapiro-NOAA** [comment]
No, this is still an issue

**Comment by uturuncoglu** [comment]
@gspetro-NOAA Yes. As I know there is no any work done to make them consistent again.

**Comment by NickSzapiro-NOAA** [comment]
Hi @uturuncoglu . Asking as you or others may already know.

We have situations in ESCOMP vs. EMC branches like Sa_u10m vs. Sa_u ; Sa_pslv vs. Sa_pbot ; Sa_t2m vs. Sa_tbot where the actual data variables really are the same but consumer component couples to a particular standard_name.

To manage "diverged" keys/variables ... I see synonym or "alias" capability in NUOPC layer in ESMF. Can that be used for allowing synonym keys? And if it can, is there downside to adding aliases to tests/parm/fd_ufs.yaml ?

**Comment by uturuncoglu** [comment]
@NickSzapiro-NOAA This is always confusing to me. In reality, Sa_u must be the lowest atmosphere and Sa_u10m must be the 10 m. But, sometimes even with the 10 meter wind Sa_u is used. I think this issue is inherited from CESM or sometime in the initial UFS WM development.

Look at following data mode (JRA-55) used by CESM:

https://github.com/ESCOMP/CDEPS/blob/3f7f22d0426ccc1428a1ebfd4357caf90009132a/datm/datm_datamode_jra_mod.F90#L266

They are setting Sa_u and Sa_v to be same with Sa_u10m and Sa_v10m. This is very confusing. It would be nice to coordinate with the CESM developers (@billsacks) and start discussion about it to come up a solution. 

Yes, aliases can be used but if you also need to pass lowest atmospheric state like Sa_u etc. along with 10 meter wind then aliases could break your application since same field name will start to point two different field.
